StandardConfig/production/HighLevelReco/HighLevelReco.xml

```diff
@@ -95,9 +95,12 @@
 
 
   <processor name="MyGammaGammaSolutionFinder" type="GammaGammaSolutionFinder">
-    <parameter name="InputParticleCollectionName1" value="GammaGammaCandidatePi0s" />
-    <parameter name="InputParticleCollectionName2" value="GammaGammaCandidateEtas" />
-    <parameter name="InputParticleCollectionName3" value="GammaGammaCandidateEtaPrimes" />
+    <parameter name="gammagammaCandidateCollections" >
```

Comment:
Is `gammagammaCandidateCollections` the correct  capitalization or the one from the Gaudi configuration with `GammaGammaCandidateCollections`?